RealEstate/apps/core/forms.py

```diff
@@ -20,3 +21,31 @@ class UserCreationForm(UserCreationForm):
     class Meta:
         model = User
         fields = ('email',)
+
+
+class addHomeForm(forms.Form):
+    """
+    Used for adding a home to a users home list
+    """
+    nickname = forms.CharField(
+        label="Nickname",
+        required=True
+        )
+    address = forms.CharField(
+        label="Address",
+        required=False
+        )
+
+class editHomeForm(forms.Form):
+    """
+    Used for adding a home to a users home list
+    """
+    edit_nickname = forms.CharField(
+        label="Nickname",
+        required=True
+        )
+    edit_address = forms.CharField(
+        label="Address",
+        required=False
+        )
+    homeId = forms.CharField(widget=forms.HiddenInput())
```

Comment:
The forms should be tweaked a bit. The address is a text field and the homeId should be an IntegerField.  To avoid redefining our data that has already been defined in models, we can use a ModelForm.  Its also a convention to capitalize classes but that wont affect behavior, so user preference.
```python
class AddHomeForm(forms.ModelForm):
    class Meta:
        model = House
        fields = ('nickname', 'address')

class EditHomeForm(forms.ModelForm):
    homeId = forms.IntegerField(widget=forms.HiddenInput())
    class Meta:
        model = House
        fields = ('nickname', 'address')
```